thrimbletrimmer/scripts/edit.js

```diff
@@ -267,27 +271,50 @@ window.addEventListener("DOMContentLoaded", async (event) => {
 				return;
 			}
 			const imageTemplate = document.getElementById("video-info-thumbnail-template").value;
-			imageElement.src = `/thumbnail/${globalStreamName}/source.png?timestamp=${imageTime}&template=${imageTemplate}`;
+			const [crop, loc] = getTemplatePosition();
+			const queryParts = [
+				`timestamp=${imageTime}`,
+				`template=${imageTemplate}`,
+				`crop=${crop.join(",")}`,
+				`location=${loc.join(",")}`,
+			];
+			imageElement.src = `/thumbnail/${globalStreamName}/source.png?${queryParts.join("&")}`;
```

Comment:
We should make sure we document somewhere that template names may not contain ampersands, unless we intend on URI-escaping them.

Reply:
We can enforce template names being limited to `[A-Za-z0-9_-]` when creating or modifying them by adding a check to `thrimshim.validate_template`.

Reply:
I've replaced all ad-hoc query string building with proper usage of `URLSearchParams()` which will automatically do correct encoding.